ID.me WebVerify SDK/IDmeWebVerify.m

```diff
@@ -51,118 +53,151 @@ + (IDmeWebVerify *)sharedInstance {
 - (id)init{
     self = [super init];
     if (self) {
+        self.keychainData = [[IDmeWebVerifyKeychainData alloc] init];
         [self clearWebViewCacheAndCookies];
     }
     
     return self;
 }
 
++ (void)initializeWithClientID:(NSString *)clientID redirectURI:(NSString *)redirectURI {
+    [[IDmeWebVerify sharedInstance] setClientID:clientID];
```

Comment:
Currently `sharedInstance` is initializing the instance. Should we require calling `initialize...` before calling `sharedInstance` and then calling the init from the `initialize`?